src/pages/organizations/create.tsx

```diff
@@ -1,53 +1,31 @@
-import { useRouter } from 'next/router';
-import { FC, FormEvent, useState } from 'react';
-import { CreateOrganizationRequest } from '../../api/client/models/CreateOrganizationRequest';
-import FormWrapper from '../../components/FormWrapper';
-import OrganizationEditor from '../../components/OrganisationEditor';
-import PageWrapper from '../../components/PageWrapper';
-import withAuth from '../../utils/withAuth';
-import apiClient from '@/src/api/client';
+import apiClient from "@/src/api/client";
+import { FC, FormEvent, useState } from "react";
+import { CreateOrganizationRequest } from "../../api/client/models/CreateOrganizationRequest";
+import FormWrapper from "../../components/FormWrapper";
+import OrganizationEditor from "../../components/OrganisationEditor";
+import PageWrapper from "../../components/PageWrapper";
+import withAuth from "../../utils/withAuth";
```

Comment:
Hier könnte man noch die path aliases einfügen. Für utils und api dann auch direkt in der tsconfig.

Reply:
Gute Idee! So ward es geschehen.